src/libs/tools/global_toolbox.c

```diff
@@ -238,8 +240,21 @@ static void _main_do_event(GdkEvent *event, gpointer data)
           }
           if(base_url)
           {
-            // TODO: try to use the currently set language
-            char *lang = "en";
+            dt_l10n_language_t *language
+                = (dt_l10n_language_t *)g_list_nth(darktable.l10n->languages, darktable.l10n->selected)->data;
+            char *lang = language->code;
+            const int number_of_supported_languages = 4;
+            const char *supported_languages[4] = { "en", "fr", "it", "es" };
+            bool is_language_supported = false;
```

Comment:
should be gboolean (not sure bool is supported on all GCC version and CLang). I could be wrong, but in dt we do use gboolean.